core/domain/activity_jobs_one_off.py

```diff
@@ -673,6 +673,15 @@ class AddMissingCommitLogsJob(jobs.BaseMapReduceOneOffJobManager):
 
         }
     }
+    # "Public" is the default value for post_commit_status for the
+    # question and skill models.
+    MODELS_NAMES_WITH_DEFAULT_COMMIT_STATUS = [
```

Comment:
MODELS_NAMES --> MODEL_NAMES, here and below

Reply:
Done.